dbus-opendtu.py

```diff
@@ -441,22 +482,32 @@ def main():
             '/Ac/L2/Energy/Forward': {'initial': None, 'textformat': _kwh},
             '/Ac/L3/Energy/Forward': {'initial': None, 'textformat': _kwh},
           }
+      
+      if dtu != 'template':
```

Comment:
Also dieser Part mit der Erzeugung der Objekte ist echt harter Tobak.
Wo kommt dieses "dtu" hier hier? ...ah 50 Zeilen weiter oben aus der config. Müsste also auch dtuvariant heißen.
Was bedeutet dieses dtuvariant im Haupt-Teil der Config eigentlich? Es kann doch verschiedene geben!?

Reply:
ist umgeändert in dtuvariant